terraform/deployments/monitoring-test/base/main.tf

```diff
@@ -0,0 +1,71 @@
+terraform {
```

Comment:
Could this file be moved to `terraform/deployments/monitoring-test/main.tf`, or is the `base` directory required?

Reply:
we could move the files in base one step up. It was just me structuring it this way to look nicer from my point of view.

Reply:
Moving the resources into one state file would make sense to me. It would mean only a single `terraform apply` would be required.

Reply:
we would still require the same number of Terraform applies.

Reply:
🤔 is there a way we could have a single TF apply, or is the grafana provider not clever enough to wait for the container to be ready?